### PR TITLE
Show error on Change Password Recovery Screen when user selecting the same question

### PR DESCRIPTION
**Pull request by amanalili**

### CODE REVIEW

File: packages/edge-login-ui-rn/src/components/screens/existingAccout/ChangeRecoveryScreen.js [hunk 1]
```diff
@@ -100,6 +100,10 @@ class ChangeRecoveryScreenComponent extends React.Component<Props, State> {
       this.state.question2 === s.strings.choose_recovery_question ||
       this.state.question1 === this.state.question2
 
+    if (this.state.question1 === this.state.question2) {
+      showError(s.strings.change_recovery_same_question)
```
Comment (thehobbit85) [inline packages/edge-login-ui-rn/src/components/screens/existingAccout/ChangeRecoveryScreen.js:104]: Don't you need to return here? 
Is showError enough so the function won't continue?

File: packages/edge-login-ui-rn/src/common/locales/en_US.js [hunk 2]
```diff
@@ -238,7 +238,9 @@ const strings = {
   notifications_permissions_ios:
     'Please enable notifications and background app refresh for account security. Edge will notify you if other devices attempt to access your account and will allow you to accept or reject access.',
   notifications_permissions_android:
-    'Please enable notifications for account security. Edge will notify you if other devices attempt to access your account and will allow you to accept or reject access.'
+    'Please enable notifications for account security. Edge will notify you if other devices attempt to access your account and will allow you to accept or reject access.',
+
```
Comment (thehobbit85) [inline packages/edge-login-ui-rn/src/common/locales/en_US.js:242]: Remove empty line

**Review by thehobbit85 — CHANGES_REQUESTED** [review]
See comments

File: packages/edge-login-ui-rn/src/components/screens/existingAccout/ChangeRecoveryScreen.js (hunk 1 above)
Comment (amanalili) [reply packages/edge-login-ui-rn/src/components/screens/existingAccout/ChangeRecoveryScreen.js:104]: That should be correct cause any errors beside that will indicated

**Review by thehobbit85 — APPROVED** [review]

**Pull request merged**